From a81d8b01e2cf343fae4b8650946e41e8c7a0ee91 Mon Sep 17 00:00:00 2001
From: Leonardo Sandoval <leonardo.sandoval.gonzalez@linux.intel.com>
Date: Mon, 2 Sep 2019 09:28:44 -0500
Subject: [PATCH 25/26] change to print %ld
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

fixes:

fftools/ffmpeg.c:5033:84: warning: format ‘%d’ expects argument of type ‘int’, but argument 4 has type ‘int64_t’ {aka ‘long int’} [-Wformat=]
 5033 |                         printf("| filter profiling | name=%s, init=%.2f ms, frame=%d, fps=%.2f\n",
      |                                                                                   ~^
      |                                                                                    |
      |                                                                                    int
      |                                                                                   %ld
 5034 |                                 ft->filter->name, (double)ft->init_working_time / 1000, frame_cnt, fps);
      |                                                                                         ~~~~~~~~~
      |                                                                                         |
      |                                                                                         int64_t {aka long int}
---
 fftools/ffmpeg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fftools/ffmpeg.c b/fftools/ffmpeg.c
index f533dfb..e8a32d1 100644
--- a/fftools/ffmpeg.c
+++ b/fftools/ffmpeg.c
@@ -5030,7 +5030,7 @@ static int transcode(void)
                 if (ft->sum_working_time > 1) {
                     double fps = (double)(frame_cnt * 1000000) / ft->sum_working_time;
                     if (fps < 10000) { //some filter delivered too big fps is not we focused
-                        printf("| filter profiling | name=%s, init=%.2f ms, frame=%d, fps=%.2f\n",
+                        printf("| filter profiling | name=%s, init=%.2f ms, frame=%ld, fps=%.2f\n",
                                 ft->filter->name, (double)ft->init_working_time / 1000, frame_cnt, fps);
                     }
                 }
-- 
2.23.0

